doc/source/release-notes/Percona-Server-5.5.45-37.4.rst

```diff
@@ -0,0 +1,20 @@
+.. rn:: 5.5.45-37.4
+
+==============================
+ |Percona Server| 5.5.45-37.4
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.45-37.4 on September 2nd, 2015. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.45-37.4/>`_ and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.45 <http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-45.html>`_, including all the bug fixes in it, |Percona Server| 5.5.45-37.4 is now the current stable release in the 5.5 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.5.45-37.4 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.5.45-37.4>`_. 
+
+Bugs Fixed
+==========
+
+ Querying ``INFORMATION_SCHEMA``. :table:`GLOBAL_TEMPORARY_TABLES` or :table:`TEMPORARY_TABLES` tables would crash threads working with temporary tables. Bug fixed :bug:`1113388`.
```

Comment:
- only GLOBAL_TEMPORARY_TABLES is affected;
- crash happens for internal temp tables used by ALTER TABLE